omero/sysadmins/server-upgrade.txt

```diff
@@ -96,6 +96,16 @@ Since 5.3, all official OMERO.web plugins can be installed from :pypi:`Python Pa
 You should remove all previously installed plugins and install the latest
 versions using `pip <https://pip.pypa.io/en/stable/>`_.
 
+Since 5.3, the `Open with` configuration allows users to open data in other web applications
+e.g. open images in a custom viewer or open images in a new figure with OMERO.figure.
+After installing OMERO.figure, run the following command to add OMERO.figure to the `Open with` options, 
```

Comment:
I'd be inclined to put "After installing OMERO.figure (or other web app), run..." here just to make doubly clear we are just using figure as the example.